src/tests/rules/sort-collections.test.ts

```diff
@@ -212,6 +239,16 @@ ruleTester.run("sort-collections", rule, {
 		"build": "echo test",
         "postbuild": "echo test"
 	}
+}`,
+		},
+		{
+			code: `{
+	"nested": {
+		"devDependencies": {
+			"typescript": "4.8.4",
+			"eslint-plugin-package-json": "0.31.0"
+		}
+	}
```

Comment:
This test ensures that nested properties having the same names as root level ones are *no longer* touched.